nest/src/auth/auth.service.ts

```diff
@@ -14,32 +15,33 @@ export class AuthService {
     private userService: UsersService,
   ) {}
 
-  generateJWT(payload: any) {
+  generateJWT(payload: JwtUserDto) {
```

Comment:
Not sure if we care about that, but we can declare this function as private for example. I have noticed other cases in this file. Just give an input, but it's up to you